### PR TITLE
Adding command line argument to modify Delphes card

### PR DESCRIPTION
**Pull request by clementhelsens**
Add delphes card as possible argument and add a generic pythia command for batch

### CODE REVIEW

File: FWCore/python/FWCore/joboptions.py [hunk 1]
```diff
@@ -4,11 +4,13 @@ def parse_standard_job_options(scriptname=""):
     """
     Returns the parsed arguments, adding a parser with commonly needed opts:
     - args.nevents      -- number of events (int), specify with --nevents
+    - args.delphescard  -- the input delphes card (string), specify with --delphescard
```
Comment (jlingema) [inline FWCore/python/FWCore/joboptions.py]: The idea was to have "standard" options here and at the same time allow parsers for specific things in the options file. E.g. like [this](https://gist.github.com/jlingema/d170c552bd3caf71a3d713ccf04d5fd5)

**Comment by clementhelsens** [comment]
Hello, 

I made the requested changes.

File: Sim/SimDelphesInterface/options/PythiaDelphes_config.py [hunk 2]
```diff
@@ -40,6 +40,12 @@ def apply_paths(obj, names):
   for attr, name in names.iteritems():
     getattr(obj.DataOutputs, attr).Path = name
 
+
+import argparse
+parser = argparse.ArgumentParser()
+parser.add_argument('--delphescard', type=str, default='', help='specify an input delphes card')
```
Comment (hegner) [inline Sim/SimDelphesInterface/options/PythiaDelphes_config.py]: if the default is an invalid value - and an empty string is - one should say default=None. I am nitpicking though.

**Comment by clementhelsens** [comment]
done!

**Pull request merged**